src/shipit_static_analysis/tests/test_clang.py

```diff
@@ -85,3 +86,15 @@ def test_clang_format(tmpdir):
 
     # At the end of the process, original file is patched
     assert bad_file.read() == BAD_CPP_VALID
+
+    # Test stats
+    stats, reporter = mock_stats
+    stats.api.flush(time.time() + 20)
+
+    metrics = reporter.get_metrics('issues.clang-format')
```

Comment:
Can we test this for clangtidy and mozlint too?

Reply:
I added a unit test for the clang-tidy runnner, but can't really do the same for mozlint as it depends on mach...